Gems/Atom/Utils/Code/Source/PngFile.cpp

```diff
@@ -25,6 +25,46 @@ namespace AZ
             {
```

Comment:
This should be under namespace specific to the PNG

Reply:
Happy to change it, but why? Isn't it enough to be in an anonymous namespace inside the .cpp?

Reply:
I agree with Mike, what utility would naming the namespace provide? These look to just be utility functions to interface with libpng error reporting and so are only likely to be used in this cpp file.